libs/pages/elewa/invest/src/lib/components/elewa-invest-investing-in-elewa-section/elewa-invest-investing-in-elewa-section.component.html

```diff
@@ -0,0 +1 @@
+<elewa-group-elewa-group-image-and-text-banner [imageURL]="image" [titleText]="titleText" [backgroundColor]="backgroundColor" [paragraphTexts]="paragraph" [imagePlacement]="imagePlacement"></elewa-group-elewa-group-image-and-text-banner>
```

Comment:
button options should be added here as well.

Reply:
Done :100: